styles/style.css

```diff
@@ -161,6 +161,37 @@ main .feed .tag {
     color: var(--main-color);
 }
 
+main .feed button {
+    border-radius: 50%;
+
+    padding: 8px 8px 3px 8px;
+}
+
+main .tweet-options button:hover,
+main .feed button:hover {
+    background-color: #1ea1f31a;
+}
+
+main .feed button svg:hover {
+    fill: var(--main-color);
+}
+
+main .feed .retweet:hover {
+    background-color: #45db1f1a;
+}
+
+main .feed .retweet svg:hover {
+    fill: #47bb2a;
+}
+
+main .feed .like:hover {
+    background-color: #f918801a;
+}
+
+main .feed .like svg:hover {
+    fill: #f91880;
```

Comment:
I am not happy with the behaviour of these svg:hover. It only changes the color of the svg if the mouse pointer is exactly over a vector. Any suggestion on how to change the color of the svg when the parent button is hovered?

Reply:
maybe `main .feed .like:hover svg`

Reply:
That works exactly as I wanted, thank you for this solution!